test/teos/e2e/test_basic_e2e.py

```diff
@@ -187,6 +209,15 @@ def test_multiple_appointments_life_cycle(run_bitcoind):
         add_appointment(appointment)
         appointments_in_watcher += 1
 
+    available_slots -= 5
+
+    # Check that get_subscription_info also detects these new appointments and returns the correct info.
+    r = get_subscription_info()
+    # We've added 6 appointments so far
+    assert len(r.get("appointments")) == 6
```

Comment:
Same here, this gets hard to maintain given it is hardcoded, there are currently two variables keeping track of how many appointments are in the watcher and the responder, this should be equal to the sum of both.